### PR TITLE
keylime example

### PR DESCRIPTION
**Pull request by Itxaka**
Signed-off-by: Itxaka <itxaka@kairos.io>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

File: examples/keylime/keylime.yaml [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+          owner_string: "keylime"
+          permissions: 0640
+        - path: /etc/keylime/agent.conf.d/10-config.conf
+          # This is the agent config, it needs to know the registrar IP
+          # If the ip is not set, the agent will listen on localhost only
+          # The UUID is the agent UUID, it needs to be unique per agent. You can leave it empty and the agent will generate one on boot
+          content: |
+            [agent]
+            ip = '0.0.0.0'
+            registrar_ip = '192.168.100.184'
```
Comment (mudler) [inline examples/keylime/keylime.yaml:74]:
```suggestion
            registrar_ip = '192.168.100.184' # change to the keylime remote attestation server IP
```

**Comment by mudler** [comment]
given the topic complexity would be probably helpful for users to have a small README on how to tie all the pieces in, and maybe a mention of it in the docs, otherwise the example by itself is going to be hard to consume

File: examples/keylime/keylime.yaml (hunk 1 above)
Comment (Itxaka) [reply examples/keylime/keylime.yaml:74]: isnt this implied in line 68?

File: examples/keylime/README.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
+# Setting up the keylime agent in Kairos
+
+Most of the steps are already covered in the [Keylime documentation](https://keylime-docs.readthedocs.io/en/latest/). Here we will cover the steps that are specific to Kairos.
+
+
+We provide thee keylime agent as a luet package for ease of installation.
```
Comment (mudler) [inline examples/keylime/README.md:6]:
```suggestion
We provide the keylime agent as a luet package for ease of installation.
```

**Review by mudler — APPROVED** [review]
That is a very good and detailed example, thanks @Itxaka !

**Pull request merged**